app/src/androidTest/java/com/github/h3lp3rs/h3lp/signIn/SignInActivitySignedInUserTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.github.h3lp3rs.h3lp.signIn
```

Comment:
I would suggest creating a package of sign in tests, rather than coming up with complex class names.
Edit: Oh but it is in a separate package! Package names should not be in camelcase! Also, since it is in the SignIn package, you can simplify the class name!